script/lib/l10n/grd_utils.py

```diff
@@ -271,6 +271,12 @@ def get_original_grd(src_root, grd_file_path):
     if grd_file_name == 'android_webapps_strings.grd':
         return os.path.join(src_root, 'components', 'webapps', 'browser',
                             'android', 'android_webapps_strings.grd')
+    if grd_file_name == 'browser_ui_strings.grd':
+        return os.path.join(src_root, 'components', 'browser_ui', 'strings',
+                            'android', 'browser_ui_strings.grd')
+    if grd_file_name == 'site_settings.grdp':
+        return os.path.join(src_root, 'components', 'browser_ui', 'strings',
+                            'android', 'site_settings.grdp')
```

Comment:
I don't think we need to add `grd`**p** files here.

Reply:
You're right this one is not required, pushed the change without it